docs/contributing.md

```diff
@@ -20,7 +30,7 @@ git clone https://github.com/opsdroid/opsdroid.git
 cd opsdroid
 
 # install the project in "editable" mode
-pip install -e .
+pip install -e.
```

Comment:
I would leave the space in, while it is optional I think it makes it more readable.